Misc/NEWS.d/next/Core and Builtins/2023-01-30-15-40-29.gh-issue-97933.nUlp3r.rst

```diff
@@ -0,0 +1,2 @@
+Inline list, dict and set comprehensions to improve performance
```

Comment:
We should also highlight the minor compatibility breaks this causes. Are you planning to add that to What's New?

Reply:
Good point, will add a more detailed paragraph to What's New including the behavior changes. If I do that, is there any reason to keep a NEWS entry also?

Reply:
I think we do still need a NEWS entry for the change, but it can stay brief; the detail can just go into the What's New.

Reply:
In https://github.com/python/cpython/pull/103348/files, I linked directly to my What's New entry from my NEWS entry -- could consider doing something similar!

Reply:
Good suggestion! In this case since there's a PEP that has even more detail than the What's New entry, I decided to have NEWS and What's New both link directly to the PEP, instead of having NEWS link to What's New.